QUEST.tsv

```diff
@@ -45,251 +45,251 @@ QUEST_20150317_000044	Items to be deleted
 QUEST_20150317_000140	Retired Hunter
-QUEST_20150317_000141	I remember this battle in a deep canyon, where soldiers fought a huge monster. {nl}It was amazing how the giant thing was turned around and there were some great spoils, too.
+QUEST_20150317_000141	I remember this battle in a deep canyon, where soldiers fought a huge monster. {nl}Seeing the giant thing fall was amazing, but what was more amazing was the spoils.
 QUEST_20150317_000142	Old Potter
-QUEST_20150317_000143	I put prayers on my earthware, one by one. {nl}That way, the pots will one day be filled with the Goddess' blessings.
+QUEST_20150317_000143	I put a prayer on my earthenware, one by one. {nl}That way, the pots will one day be filled with the Goddess' blessings.
 QUEST_20150317_000144	Swordsmen should have Strength, but you can also have Dexterity and Constitution. {nl}You'll need to think about what to start with.
-QUEST_20150317_000145	Wizards work with Intelligence, but shouldn't forget Spirit for more spellcasting. {nl}You'll have to choose between being balanced or being focused.
+QUEST_20150317_000145	Wizards work with Intelligence, but shouldn't forget Spirit for more spell-casting. {nl}You'll have to choose between being balanced or specializing.
 QUEST_20150317_000146	Archers should have Strength, but Dexterity will prevent problems with movement. {nl}You'll need to think about what to start with.
-QUEST_20150317_000147	Clerics work with Intelligence, but might need a lot of Strength or Spirit. {nl}You should really decide what you want first.
+QUEST_20150317_000147	Clerics work with Intelligence, but might need a lot of Strength or Spirit. {nl}You should really decide what you want, first.
 QUEST_20150317_000148	Doppelsoeldner Master
-QUEST_20150317_000149	The things that are worthwhile will always carry a degree of risk. {nl}Naturally, for big rewards, you'll have to put up with danger.
+QUEST_20150317_000149	Worthwhile things always carry a degree of risk. {nl}Naturally, for the big rewards, you'll have to put up with lots of danger.
 QUEST_20150317_000150	Corsair Master
-QUEST_20150317_000151	I won't help anyone in trouble if there isn't anything in it for me.
+QUEST_20150317_000151	If there isn't anything in it for me, I won't lift a finger.
 QUEST_20150317_000152	Boulder
 QUEST_20150317_000153	A large boulder is blocking the way.
 QUEST_20150317_000154	It needs to be repaired.
 QUEST_20150317_000155	A boulder blocks the way.
 QUEST_20150317_000156	Magical Research Material
-QUEST_20150317_000157	It appears to be someone's research notes from the Mage Tower. {nl}Should it be copied?
-QUEST_20150317_000158	Not everything can be done with hiding and sneaking. {nl}A true Scout must also be ready for a fight.
-QUEST_20150317_000159	You'll need a lot of little things while you're travelling. {nl}You should buy up your provisions in advance.
+QUEST_20150317_000157	It appears to be someone's research notes from Magicians' Tower. {nl}Should it be copied?
```

Comment:
https://github.com/ttgmichael/EnglishTranslation/wiki/2.2.%20Map

*(The dedicated contributors use this Wiki as an updated resource)*

Magicians' Tower -> Mage Tower